lib/Pages/ListPage.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart';
+import 'dart:convert';
+import 'package:list_maker/API/api_requests.dart';
+import 'package:list_maker/Widgets/add_item_widget.dart';
+import 'package:list_maker/Widgets/items_list_widget.dart';
+
+class NewList extends StatefulWidget {
+  final user;
+  String list_name = "";
+  List uncheckedItems = [];
+  List checkedItems = [];
+
+  NewList(this.user, this.list_name, this.uncheckedItems,
```

Comment:
What does this line do? What does this comment mean?

Reply:
this line is the constructor of the class. the comment is the previous constructor that i forgot to delete